### PR TITLE
Fix #50: logo not visible in package

### LINKED ISSUE #50: logo not visible in renderwindow after packaging
logo not visible in renderwindow after packaging

image

### PR DESCRIPTION
**Pull request by dboun**
Fix #50

### CODE REVIEW

**Comment by dboun** [comment]
(wait)

**Comment by dboun** [comment]
Didn't actually fix it

File: Plugins/org.mitk.cbica.captk.ui/src/QmitkExtWorkbenchWindowAdvisor.cpp [hunk 1]
```diff
@@ -824,6 +824,12 @@ void QmitkExtWorkbenchWindowAdvisor::PostWindowCreate()
     berry::IPreferences::Pointer stylePrefs = prefService->GetSystemPreferences()->Node(berry::QtPreferences::QT_STYLES_NODE);
     bool showCategoryNames = stylePrefs->GetBool(berry::QtPreferences::QT_SHOW_TOOLBAR_CATEGORY_NAMES, true);
 
+    // CaPTk logo
+    prefService->GetSystemPreferences()->Node("org.mitk.editors.stdmultiwidget")->Put(
+      "DepartmentLogo",
```
Comment (ashishsingh18) [inline Plugins/org.mitk.cbica.captk.ui/src/QmitkExtWorkbenchWindowAdvisor.cpp:829]: @dboun what happened when you added this?

Comment (dboun) [reply Plugins/org.mitk.cbica.captk.ui/src/QmitkExtWorkbenchWindowAdvisor.cpp:829]: Nothing changed. But maybe I put it in the wrong place?

**Pull request closed without merging**